awswrangler/_data_types.py

```diff
@@ -326,7 +326,7 @@ def athena2pyarrow(dtype: str) -> pa.DataType:  # pylint: disable=too-many-retur
         return pa.list_(value_type=athena2pyarrow(dtype=orig_dtype[6:-1]), list_size=-1)
     if dtype.startswith("struct") is True:
         return pa.struct(
-            [(f.split(":", 1)[0], athena2pyarrow(f.split(":", 1)[1])) for f in _split_struct(orig_dtype[7:-1])]
```

Comment:
When parsing the schema for `struct`, the values of all but the first key are prefixed by a single space. This is causing problems later on when we try to cast types, as `{"a": "int", "b": "string"}` cannot be cast to `{"a": "int", " b": "string"}`